test_conformance/CMakeLists.txt

```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
+
+#copy all csv files
+file(GLOB CSV_FILES
+ 	 "../test_conformance/*.csv"
```

Comment:
It would be better to avoid this use of relative paths.

Reply:
ok , I'll fix it